ai_ta_backend/vector_database.py

```diff
@@ -21,10 +21,13 @@
 import requests
 import supabase
 from bs4 import BeautifulSoup
+

```

Comment:
sorry, can we remove unused document loaders? Memory is actually really expensive on Railway and removing imports helps a ton to save cost.

Reply:
Removed UnstructuredFileLoader